src/shared/config/ftpUrls.ts

```diff
@@ -80,8 +79,8 @@ export const getUniprotkbFtpFilenameAndUrl = (
   downloadUrl: string,
   format: FileFormat
 ) => {
-  const parsed = queryString.parseUrl(downloadUrl);
-  const { query } = parsed.query;
+  const sp = new URLSearchParams(downloadUrl);
+  const query = sp.get('query');
   const q = Array.isArray(query) ? query[0] : query;
```

Comment:
I'd say it's safe to remove this check now, because `get` will only give you a string value corresponding to the first value in the search string (in contrast with `getAll`)